_layouts/default.html

```diff
@@ -22,27 +22,33 @@
   <!-- 🥰 https://metatags.io/  -->
 
   <!-- Primary Meta Tags -->
-  <title>
+
     {% if page.hack_number %}
-      Hack {{page.hack_number | prepend: '0' | slice: -2, 2 }} – {{page.date | date: '%B %Y' }}
+      {% capture derivedTitle %}
+        Hack {{page.hack_number | prepend: '0' | slice: -2, 2 }} – {{page.date | date: '%B %Y' }}
+      {% endcapture %}
     {% else %}
-      {{ title }}
+      {% capture derivedTitle %}
+        {{ title }}
+      {% endcapture %}
     {% endif %}
+  <title>
```

Comment:
When it comes to using the `derivedTitle`, you could just:

```liquid
{% assign derivedTitle = derivedTitle | strip %}
```

so you don't need to call `strip` every time it's used

Reply:
excellent suggestion! I shall apply the latter